src/Sema.zig

```diff
@@ -6446,7 +6446,15 @@ fn analyzeCall(
             .extern_fn => return sema.fail(block, call_src, "{s} call of extern function", .{
                 @as([]const u8, if (is_comptime_call) "comptime" else "inline"),
             }),
-            else => unreachable,
+            else => {
+                if (callee_ty.isPtrAtRuntime()) {
```

Comment:
It think it would be better to remove this branch and consider all other values to be some sort of function pointers and then correct it later if that's not the case. Or just converting it to an assertion would be fine too.

Reply:
Sounds good. Do I have to do that correction too or is that already done? If it is already done, what I just pushed might be what you want.